data_structures/linked_list/multiple_linked_list.c

```diff
@@ -0,0 +1,106 @@
+#include<stdio.h>

```

Comment:
This repository is purely for educational purposes therefore kindly add a description of your code and some sample test cases for better understanding of users, use comment lines.

Reply:
👍 Moreover, this is a good C program and not a C++ program and hence should be pulled to https://github.com/TheAlgorithms/C

Reply:
@kvedala I have pulled to https://github.com/TheAlgorithms/C itself.

Reply:
> @kvedala I have pulled to https://github.com/TheAlgorithms/C itself.

Oh yes, my bad 😄

Reply:
> This repository is purely for educational purposes therefore kindly add a description of your code and some sample test cases for better understanding of users, use comment lines.

Please add documentation following doxygen format like [this](https://github.com/kvedala/C/blob/master/numerical_methods/durand_kerner_roots.c). This enables the automation tools to better generate the documentation like [this](https://kvedala.github.io/C/da/d38/durand__kerner__roots_8c.html)
Thank you